packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -355,6 +358,17 @@ class InkResponse extends StatelessWidget {
   /// Called when the user long-presses on this part of the material.
   final GestureLongPressCallback? onLongPress;
 
+  /// Called when there is a tap with the secondary button.
```

Comment:
Maybe for consistency: `Called when there is a tap with the secondary button on this part of the material`.